source/_footer.erb

```diff
@@ -0,0 +1,67 @@
+<div class="footer">
+  <div class="footer-container">
+    <div class="footer-container__address-tile">
+      <p class="footer-container__company-name">
+        unboxed
+      </p>
+      <p>hello@unboxed.co</p>
+      <div class="social-icons">
+        <a class="social-icons__icon-container" href="http://twitter.com/Ubxd" target="_blank">
+          <div class="social-icons__icon--twitter">
+            <%= image_tag 'twitter.png', class: 'social-icons__image', alt: 'Twitter' %>
+          </div>
+        </a>
+        <a class="social-icons__icon-container" href="http://www.linkedin.com/company/unboxed-consulting" target="_blank">
+          <div class="social-icons__icon--linkedin">
+            <%= image_tag 'linkedin.png', class: 'social-icons__image', alt: 'LinkedIn' %>
+          </div>
+        </a>
+      </div>
+      <div class="footer-container__registration-address">
+        <p>registered in England and Wales</p>
+        <p>05440833</p>
```

Comment:
_Maybe_ we could link to https://beta.companieshouse.gov.uk/company/05440833 :stuck_out_tongue: